installers/container-sirepo-base/radiasoft-download.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+container_sirepo_base_main() {
+    declare user=${1:-}
+    if [[ $user == root ]]; then
+        container_sirepo_base_root
+        return
+    fi
+    container_sirepo_base_run_user
+}
+
+container_sirepo_base_root() {
+    build_yum install fedora-workstation-repositories
+    build_yum config-manager --set-enabled google-chrome
+    build_yum install google-chrome-stable
+}
+
+container_sirepo_base_run_user() {
+    install_url radiasoft/sirepo
+    #POSIT: This relies on the fact that individual package names don't have spaces or special chars
+    npm install -g \
+        $(install_download package.json | jq -r '.devDependencies | to_entries | map("\(.key)@\(.value)") | .[]')
```

Comment:
Do you need to include package.json in this repo?

Reply:
Copy paste bug. Removed from container-sirepo-devbox https://github.com/radiasoft/container-sirepo-devbox/pull/8/commits/3d3d94043bb4ca2c3f69441fbe99b1500cd076e3